### ISSUE TITLE
FAILED: DEVBOT: neonKUBE build/test failed

### ISSUE THREAD

**Issue by neonforgedevbot** [nforgeio/neonKUBE#1266]
<table>
<tr>
  <td><b>Outcome:</b></td>
  <td><b>TESTS FAILED</b></td>
</tr>
<tr>
  <td><b>Branch:</b></td>
  <td>master</td>
</tr>
<tr>
  <td><b>Config:</b></td>
  <td>debug</td>
</tr>
<tr>
  <td><b>Filter:</b></td>
  <td>Category!=buggy&Category!=investigate</td>
</tr>
<tr>
  <td><b>Commit:</b></td>
  <td>-na-</td>
</tr>
<tr>
  <td><b>Runner:</b></td>
  <td>WRT-00-GITHUB-RUNNER-01</td>
</tr>
<tr>
  <td><b>Workflow Run:</b></td>
  <td><a href="https://github.com/nforgeio/neonCLOUD/actions/runs/1231844223">link</a></td>
</tr>
<tr>
  <td><b>Workflow:</b></td>
  <td><a href="https://github.com/nforgeio/neonCLOUD/blob/master/.github/workflows/neonkube-tester.yaml">link</a></td>
</tr>
<tr>
  <td><b>Test.Neon.Cassandra:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Cassandra-net48.md">details</a> - 00:01:01 pass: <b>16</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Cassandra:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Cassandra-net5.0.md">details</a> - 00:01:04 pass: <b>16</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Cassandra:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Cassandra-netcoreapp3.1.md">details</a> - 00:00:59 pass: <b>16</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Common:</b></td>
  <td>&#x26A0 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Common-net48.md">details</a> - 00:03:36 pass: <b>619</b> fail: <b>0</b> skipped: 23</td>
</tr><tr>
  <td><b>Test.Neon.Common:</b></td>
  <td>&#x26A0 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Common-net5.0.md">details</a> - 00:03:38 pass: <b>619</b> fail: <b>0</b> skipped: 23</td>
</tr><tr>
  <td><b>Test.Neon.Common:</b></td>
  <td>&#x26A0 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Common-netcoreapp3.1.md">details</a> - 00:03:43 pass: <b>619</b> fail: <b>0</b> skipped: 23</td>
</tr><tr>
  <td><b>Test.Neon.Couchbase:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Couchbase-net5.0.md">details</a> - 00:03:20 pass: <b>8</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Couchbase:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Couchbase-netcoreapp3.1.md">details</a> - 00:03:45 pass: <b>8</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Cryptography:</b></td>
  <td>&#x26A0 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Cryptography-net48.md">details</a> - 00:00:04 pass: <b>46</b> fail: <b>0</b> skipped: 2</td>
</tr><tr>
  <td><b>Test.Neon.Cryptography:</b></td>
  <td>&#x26A0 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Cryptography-net5.0.md">details</a> - 00:00:04 pass: <b>46</b> fail: <b>0</b> skipped: 2</td>
</tr><tr>
  <td><b>Test.Neon.Cryptography:</b></td>
  <td>&#x26A0 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Cryptography-netcoreapp3.1.md">details</a> - 00:00:05 pass: <b>46</b> fail: <b>0</b> skipped: 2</td>
</tr><tr>
  <td><b>Test.Neon.Deployment:</b></td>
  <td>&#x26A0 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Deployment-net5.0.md">details</a> - 00:03:03 pass: <b>133</b> fail: <b>0</b> skipped: 1</td>
</tr><tr>
  <td><b>Test.Neon.Identity:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Identity-net5.0.md">details</a> - 00:01:20 pass: <b>1</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Kube:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Kube-net5.0.md">details</a> - 00:00:10 pass: <b>16</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.ModelGen:</b></td>
  <td>&#x274C <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.ModelGen-net5.0.md">details</a> - 00:02:17 pass: <b>41</b> fail: <b>48</b> skipped: 4</td>
</tr><tr>
  <td><b>Test.Neon.ModelGenCli:</b></td>
  <td>&#x274C <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.ModelGenCli-net5.0.md">details</a> - 00:00:09 pass: <b>0</b> fail: <b>7</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Postgres:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Postgres-net48.md">details</a> - 00:05:41 pass: <b>20</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Postgres:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Postgres-net5.0.md">details</a> - 00:05:45 pass: <b>20</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Postgres:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Postgres-netcoreapp3.1.md">details</a> - 00:05:42 pass: <b>20</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Service:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Service-net5.0.md">details</a> - 00:01:26 pass: <b>34</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Service:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Service-netcoreapp3.1.md">details</a> - 00:01:27 pass: <b>34</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Temporal:</b></td>
  <td>&#x274C <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Temporal-net48.md">details</a> - 00:00:52 pass: <b>0</b> fail: <b>10</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Temporal:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Temporal-net5.0.md">details</a> - 00:00:49 pass: <b>10</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Temporal:</b></td>
  <td>&#x274C <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Temporal-netcoreapp3.1.md">details</a> - 00:00:55 pass: <b>0</b> fail: <b>10</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Web:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Web-net5.0.md">details</a> - 00:00:03 pass: <b>2</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Web:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Web-netcoreapp3.1.md">details</a> - 00:00:03 pass: <b>2</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Xunit:</b></td>
  <td>&#x274C <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Xunit-net5.0.md">details</a> - 00:05:19 pass: <b>47</b> fail: <b>2</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Xunit:</b></td>
  <td>&#x274C <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.Xunit-netcoreapp3.1.md">details</a> - 00:05:16 pass: <b>47</b> fail: <b>2</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.YugaByte:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.YugaByte-net48.md">details</a> - 00:03:27 pass: <b>8</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.YugaByte:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.YugaByte-net5.0.md">details</a> - 00:03:22 pass: <b>8</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.YugaByte:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.Neon.YugaByte-netcoreapp3.1.md">details</a> - 00:03:20 pass: <b>8</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.NeonCli:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T03_59_41Z-Test.NeonCli-net5.0.md">details</a> - 00:00:19 pass: <b>24</b> fail: <b>0</b> skipped: 0</td>
</tr>
</table>

**Comment by neonforgedevbot** [comment]
<table>
<tr>
  <td><b>Outcome:</b></td>
  <td><b>TESTS FAILED</b></td>
</tr>
<tr>
  <td><b>Branch:</b></td>
  <td>master</td>
</tr>
<tr>
  <td><b>Config:</b></td>
  <td>debug</td>
</tr>
<tr>
  <td><b>Filter:</b></td>
  <td>Category!=buggy&Category!=investigate</td>
</tr>
<tr>
  <td><b>Commit:</b></td>
  <td>-na-</td>
</tr>
<tr>
  <td><b>Runner:</b></td>
  <td>WRT-00-GITHUB-RUNNER-01</td>
</tr>
<tr>
  <td><b>Workflow Run:</b></td>
  <td><a href="https://github.com/nforgeio/neonCLOUD/actions/runs/1232779811">link</a></td>
</tr>
<tr>
  <td><b>Workflow:</b></td>
  <td><a href="https://github.com/nforgeio/neonCLOUD/blob/master/.github/workflows/neonkube-tester.yaml">link</a></td>
</tr>
<tr>
  <td><b>Test.Neon.Cassandra:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Cassandra-net48.md">details</a> - 00:01:03 pass: <b>16</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Cassandra:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Cassandra-net5.0.md">details</a> - 00:01:01 pass: <b>16</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Cassandra:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Cassandra-netcoreapp3.1.md">details</a> - 00:01:00 pass: <b>16</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Common:</b></td>
  <td>&#x26A0 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Common-net48.md">details</a> - 00:03:40 pass: <b>619</b> fail: <b>0</b> skipped: 23</td>
</tr><tr>
  <td><b>Test.Neon.Common:</b></td>
  <td>&#x26A0 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Common-net5.0.md">details</a> - 00:03:38 pass: <b>619</b> fail: <b>0</b> skipped: 23</td>
</tr><tr>
  <td><b>Test.Neon.Common:</b></td>
  <td>&#x274C <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Common-netcoreapp3.1.md">details</a> - 00:03:30 pass: <b>618</b> fail: <b>1</b> skipped: 23</td>
</tr><tr>
  <td><b>Test.Neon.Couchbase:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Couchbase-net5.0.md">details</a> - 00:03:29 pass: <b>8</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Couchbase:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Couchbase-netcoreapp3.1.md">details</a> - 00:03:27 pass: <b>8</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Cryptography:</b></td>
  <td>&#x26A0 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Cryptography-net48.md">details</a> - 00:00:05 pass: <b>46</b> fail: <b>0</b> skipped: 2</td>
</tr><tr>
  <td><b>Test.Neon.Cryptography:</b></td>
  <td>&#x26A0 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Cryptography-net5.0.md">details</a> - 00:00:04 pass: <b>46</b> fail: <b>0</b> skipped: 2</td>
</tr><tr>
  <td><b>Test.Neon.Cryptography:</b></td>
  <td>&#x26A0 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Cryptography-netcoreapp3.1.md">details</a> - 00:00:04 pass: <b>46</b> fail: <b>0</b> skipped: 2</td>
</tr><tr>
  <td><b>Test.Neon.Deployment:</b></td>
  <td>&#x26A0 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Deployment-net5.0.md">details</a> - 00:03:24 pass: <b>133</b> fail: <b>0</b> skipped: 1</td>
</tr><tr>
  <td><b>Test.Neon.Identity:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Identity-net5.0.md">details</a> - 00:01:21 pass: <b>1</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Kube:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Kube-net5.0.md">details</a> - 00:00:09 pass: <b>16</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.ModelGen:</b></td>
  <td>&#x274C <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.ModelGen-net5.0.md">details</a> - 00:02:16 pass: <b>41</b> fail: <b>48</b> skipped: 4</td>
</tr><tr>
  <td><b>Test.Neon.ModelGenCli:</b></td>
  <td>&#x274C <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.ModelGenCli-net5.0.md">details</a> - 00:00:09 pass: <b>0</b> fail: <b>7</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Postgres:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Postgres-net48.md">details</a> - 00:05:35 pass: <b>20</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Postgres:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Postgres-net5.0.md">details</a> - 00:05:39 pass: <b>20</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Postgres:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Postgres-netcoreapp3.1.md">details</a> - 00:05:38 pass: <b>20</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Service:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Service-net5.0.md">details</a> - 00:01:36 pass: <b>34</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Service:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Service-netcoreapp3.1.md">details</a> - 00:01:30 pass: <b>34</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Temporal:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Temporal-net48.md">details</a> - 00:00:49 pass: <b>10</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Temporal:</b></td>
  <td>&#x274C <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Temporal-net5.0.md">details</a> - 00:00:55 pass: <b>0</b> fail: <b>10</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Temporal:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Temporal-netcoreapp3.1.md">details</a> - 00:00:48 pass: <b>10</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Web:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Web-net5.0.md">details</a> - 00:00:03 pass: <b>2</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Web:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Web-netcoreapp3.1.md">details</a> - 00:00:03 pass: <b>2</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Xunit:</b></td>
  <td>&#x274C <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Xunit-net5.0.md">details</a> - 00:05:15 pass: <b>47</b> fail: <b>2</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.Xunit:</b></td>
  <td>&#x274C <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.Xunit-netcoreapp3.1.md">details</a> - 00:05:09 pass: <b>47</b> fail: <b>2</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.YugaByte:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.YugaByte-net48.md">details</a> - 00:03:23 pass: <b>8</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.YugaByte:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.YugaByte-net5.0.md">details</a> - 00:03:21 pass: <b>8</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.Neon.YugaByte:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.Neon.YugaByte-netcoreapp3.1.md">details</a> - 00:03:19 pass: <b>8</b> fail: <b>0</b> skipped: 0</td>
</tr><tr>
  <td><b>Test.NeonCli:</b></td>
  <td>&#x2714 <a href="https://github.com/nforgeio/artifacts/blob/master/test/2021-09-14T10_44_04Z-Test.NeonCli-net5.0.md">details</a> - 00:00:18 pass: <b>24</b> fail: <b>0</b> skipped: 0</td>
</tr>
</table>

**Comment by jefflill** [comment]
I think I've addressed most of these.

The `ModelGen` related tests are having a lot of trouble building when run via:
```
dotnet test ...
```
I believe this may be due to the fact that I wasn't passing **--output** to **dotnet test** which means that the test build output folder won't be the same as for the full solution build.  This also means that the output folders are not created under **$/bin** which means that they're not automatically git-ignored.  These folders will be wiped when the next workflow runs, but behavior is still icky.

I modified the test action to pass trhe standard output directory to **dotnet test**
